lib/sqlitex/statement.ex

```diff
@@ -170,35 +172,36 @@ defmodule Sqlitex.Statement do
   ## Parameters
 
   * `statement` - The statement to run.
+  * `timeout` - The query timeout to be passed to esqlite.
   * `into` - The collection to put the results into. Defaults to an empty list.
 
   ## Returns
 
   * `{:ok, results}`
   * `{:error, error}`
   """
-  def fetch_all(statement, into \\ []) do
-    case raw_fetch_all(statement) do
+  def fetch_all(statement, timeout, into \\ []) do
```

Comment:
could you add another clause here to the tune of:

```elixir
def fetch_all(statement, timeout \\ @default_timeout, into \\ [])
```
So it doesn't break the existing API as bad?

Reply:
Sure